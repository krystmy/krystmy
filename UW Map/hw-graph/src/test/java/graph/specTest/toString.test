# Test to string of non empty graph

# Create a graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

# Print the toString of graph
ListNodes graph1
