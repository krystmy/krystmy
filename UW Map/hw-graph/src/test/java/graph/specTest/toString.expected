# Test to string of non empty graph

# Create a graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

# Print the toString of graph
graph1 contains: n1 n2 n3 n4